docs/node-mixin/dashboards/node.libsonnet

```diff
@@ -16,25 +16,30 @@ local gauge = promgrafonnet.gauge;
           datasource='$datasource',
           span=6,
           format='percentunit',
-          max=100,
+          max=1,
           min=0,
+          stack=true,
         )
         .addTarget(prometheus.target(
-          // TODO: Consider using `${__interval}` as range and a 1m min step.
           |||
-            1 - rate(node_cpu_seconds_total{%(nodeExporterSelector)s, mode="idle", instance="$instance"}[1m])
+            (
+              (1 - rate(node_cpu_seconds_total{%(nodeExporterSelector)s, mode="idle", instance="$instance"}[$__interval]))
+            / ignoring(cpu) group_left
+              count without (cpu)( node_cpu_seconds_total{job="default/node-exporter", mode="idle", instance="$instance"})
```

Comment:
```suggestion
              count without (cpu)( node_cpu_seconds_total{%(nodeExporterSelector)s, mode="idle", instance="$instance"})
```

Reply:
Ugh, yeah, at least you can see that I tried out that query in our setup first...